### ISSUE TITLE
Dashboard lazy loading does not interact well with multiple screen readers

### ISSUE THREAD

**Issue by dwillemv** [grafana/grafana#63392]
**Steps to reproduce**:
Open any dashboard that does not display fully in either firefox or chrome on either Windows or linux.
**Actual Result**:
A partially rendered dashboard that does not scroll or update as a screen reader user navigates the page.
I suspect (but haven't confirmed) that the page doesn't detect screen reader navigation.
I've found https://github.com/grafana/grafana/issues/17257, but what ever changed there didn't fix my issue.
**Expected Result**
Panels are rendered as a screen reader user moves over the page.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 9.3.2
- Data source type & version:
- User OS & Browser: Windows, linux. Chrome firefox
- Others: NVDA on windows, Orca on linux

**Comment by joshhunt** [comment]
Are you able to clarify what you mean by "a screen reader user navigates the page"? Specifically, how are you navigating the page, or triggering scrolls, that interacts poorly with screen readers? Is the poor interaction that the page doesn't scroll at all? Or that panels 'below the fold' are not loaded/queried?

_Without_ a screen reader active, I noticed inconsistent behaviour when tabbing through a dashboard (with many panels organised with rows) - it will tab through the `[i]` panel description button for the first group of panels, but aftwards it would only tab through each row.

I suspect that as focus is at the bottom of the viewport, panels immediately below the viewport are not rendered (and cannot be tabbed-to), but Rows are always rendered, and thus are tabbed-to (skipping all the panels in between)

**Comment by grafanabot** [comment]
We've closed this issue since it needs more information and hasn't had any activity recently. We can re-open it after you you add more information. To avoid having your issue closed in the future, please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/main/CONTRIBUTING.md) guidelines.

Happy graphing!
